mongodb-js/compass
# Backport COMPASS-3118 to 1.16 from #1550

From #1550

* COMPASS-3118: Fix orphaned windows

* :zap: Make travis use latest npm

* Try rebuilding entire package-lock for travis

* COMPASS-3118: Can no longer repro exception

* Shouldn't need to flush npm cache anymore

* COMPASS-3118: Handle potential race of COMPASS-3101

* debugging travis

* COMPASS-3113: Use xvfb-maybe instead of travis before_script

* Make travis use `npm ci` which is faster than `npm install` for actual CI

- Now that we're on `npm@6`, we can take advantage of https://docs.npmjs.com/cli/ci
- Remove `ci` run script as it conflicts with this new feature (I was ahead of my time...)
- Remove `mongodb-js-fmt` as everyone now uses prettier in their editor's

* debugging travis

* :zap: Cache npm build cache

* Revert "COMPASS-3113: Use xvfb-maybe instead of travis before_script"

This reverts commit d39c6cbd877f502f575c157a19f1054adce6df9f.

* Fix databases test

* Debugging test failures

* Fix Windows on Evergreen to use npm@6
